ci-operator/config/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-master.yaml

```diff
@@ -44,15 +47,22 @@ tests:
     make lint
   container:
     clone: true
-    from: golangci-lint-shellcheck
+    from: test-bin
```

Comment:
Just to make sure I understand the PR: we are now counting on the Makefile to install `golangci-lint`, is that correct?

Reply:
This change runs `make lint` in an image that also has Helm. Would it make sense to have two images? One that has ShellCheck, and one that has Helm?

Reply:
> Just to make sure I understand the PR: we are now counting on the Makefile to install golangci-lint, is that correct?

Yes.

> Would it make sense to have two images? One that has ShellCheck, and one that has Helm?

Originally, I thought doing it that way, but then we need to "maintain" 2 images. Tomorrow we will have another dependency and then what? we create a new image again? I rather have one `test-bin` image for all dependency for all the jobs in the CI.

When a new developer will come it will be clear he just should add dependency to the image build and not search around the doc like we did.

Reply:
Okay, I don't have a strong opinion on this to be honest. I generally prefer that each job only has the minimal set of dependencies, but if images are not built in parallel that might add a lot of overhead. I'll leave the decision to you on that one.

Reply:
I kept that one as `test-bin`